lib/common.js

```diff
@@ -764,6 +761,36 @@ function readApp(projectId, appId, cb) {
   }
 }
 
+function deleteApp(projectId, appId, cb) {
```

Comment:
Thanks. This feels like possible improvement before merging this back. Leaving that for you.